core/domain/stats_domain.py

```diff
@@ -616,12 +690,27 @@ def __init__(self, answer_occurrences=None):
             answer_occurrences if answer_occurrences else [])
 
     def to_raw_type(self):
+        """Returns the answer occurrences list in its raw type.
+
+        Returns:
+            list. The answer occurrences list in raw type.
+        """
         return [
             answer_occurrence.to_raw_type()
             for answer_occurrence in self.answer_occurrences]
 
     @classmethod
     def from_raw_type(cls, answer_occurrence_list):
+        """Returns domain object that represents an output list of
```

Comment:
Returns a domain object ...

That said, isn't this more of a "conversion"/"creation" method? Something like "Creates a domain object, given ..." Just saying what the return value doesn't seem to capture the intent fully.

Reply:
Done!